html/public/templatePublic.twig

```diff
@@ -191,21 +191,21 @@
                         </li>
                         {% if INSTANCE.publicData.enableAssets %}
                             <li class="nav-item">
-                                <a class="nav-link" href="assets.php">
+                                <a class="nav-link" href="{{ INSTANCE.publicData.customDomains[0] }}/assets.php">
```

Comment:
Has this been tested with multiple custom domains? AFAIK this is always going to redirect to the first custom domain but the user could be accessing from a different domain which is also valid

Reply:
In the event of multiple custom domains, does it matter if they get redirected to the first domain?